microsoft-edge/devtools-guide-chromium/whats-new/whats-new.md

```diff
@@ -23,22 +23,4 @@ ms.date: 05/16/2022
 * [What's New in DevTools (Microsoft Edge 102)](2022/05/devtools-102.md)
```

Comment:
change line 2 from:
```
title: What's new in Microsoft Edge DevTools
```
to:
```
title: What's New in Microsoft Edge DevTools
```

Reply:
fixed

Reply:
fix not appearing, re-entering a comment